lib/fastlane/plugin/appcenter/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module Appcenter
-    VERSION = "1.1.0"
+    VERSION = "1.1.1"
```

Comment:
Please make sure you're on recent master - we already have `1.1.1` plugin version.
Also we usually incrementing version as part of release process, so it is not required to add that in this PR

Reply:
Between the moment I started work on this and now the version has been updated :)
Thanks for the notice, I merged master to resolve this.